tardis/default_settings/auth.py

```diff
@@ -32,6 +32,10 @@
 AUTHENTICATION_BACKENDS = (
     'django.contrib.auth.backends.ModelBackend',
     'tardis.tardis_portal.auth.authorisation.ACLAwareBackend',
+    'social_core.backends.open_id.OpenIdAuth',
```

Comment:
We should discuss whether the social_core authentication backends really need to be in MyTardis's default_settings.  They could be included but commented out.

We also need to add a comment that importing `social_django/__init__.py` is needed so that the authentication backend's init signature matches what Django expects.

From social_django/__init__.py:
```
# django.contrib.auth.load_backend() will import and instanciate the
# authentication backend ignoring the possibility that it might
# require more arguments. Here we set a monkey patch to
# BaseAuth.__init__ to ignore the mandatory strategy argument and load
# it.
```